.github/workflows/ci_tests.yml

```
@@ -46,24 +46,25 @@ jobs:
           - description: "ASan"
             args: "-DCMAKE_CXX_FLAGS=-fsanitize=address -fsanitize=undefined"
         include:
-          - platform: ubuntu-latest
+          - platform: ubuntu-22.04
             compiler:
               cpp: g++
               c: gcc
-            cpp_version: 17
+            cpp_version: 20
             cmake_args:
               description: "Werror"
               cmake_args: "-DCMAKE_CXX_FLAGS='-Werror=all -Werror=extra'"
-          - platform: ubuntu-latest
-            compiler:
-              cpp: g++
-              c: gcc
-            cpp_version: 17
-            cmake_args:
-              description: "Dynamic"
-              cmake_args: "-DBUILD_SHARED_LIBS=on"
+          # NOTE: This example is an header only library!
+          # - platform: ubuntu-22.04
+          #   compiler:
+          #     cpp: g++
+          #     c: gcc
+          #   cpp_version: 20
+          #   cmake_args:
+          #     description: "Dynamic"
+          #     cmake_args: "-DBUILD_SHARED_LIBS=on"
 
-    name: "Bulid & Test: ${{ matrix.compiler.c }} ${{ matrix.cpp_version }} ${{ matrix.cmake_args.description }}"
+    name: "Test on ${{ matrix.platform }}: ${{ matrix.compiler.c }} ${{ matrix.cpp_version }} ${{ matrix.cmake_args.description }}"
```

```suggestion
    name: "${{ matrix.platform }}: ${{ matrix.compiler.c }} ${{ matrix.cpp_version }} ${{ matrix.cmake_args.description }}"
```

Same here: It's enough to see `CI Tests / ubuntu-24.04: gcc-14 c++17 ..."